WebRoot/js/zimbraMail/calendar/view/ZmApptQuickAddDialog.js

```diff
@@ -139,42 +114,26 @@ function() {
 
 	// save field values of this view w/in given appt
 	appt.setName(this._subjectField.getValue());
-	appt.freeBusy = this._showAsSelect.getValue();
-	appt.privacy = this._privacySelect.getValue();
 	var calId = this._folderSelect.getValue();
 	appt.setFolderId(calId);
 	appt.setOrganizer(this._calendarOrgs[calId]);
 
 	// set the start date by aggregating start date/time fields
-	var startDate = AjxDateUtil.simpleParseDateStr(this._startDateField.value);
-	var endDate = AjxDateUtil.simpleParseDateStr(this._endDateField.value);
+	var startDate = this._appt.startDate;
```

Comment:
Seems like we are using `this._appt` object now to get dates and before we was using form input fields to get that value.  So will this `this._appt` will also be initialized for creating new appointment?

Reply:
Adding to this, 
Can there be a case when this._appt is not defined?

Reply:
it will be always defined as in any case we are selecting a timeslot before opening a quick add dialog so it will create appt object